packages/next/build/swc/src/next_dynamic.rs

```
@@ -0,0 +1,174 @@
+                        type_args: None,
+                      })),
+                      spread: None,
+                    })],
+                    span: DUMMY_SP,
+                  }))),
+                  is_async: false,
+                  is_generator: false,
+                  span: DUMMY_SP,
+                  return_type: None,
+                  type_params: None,
+                })),
+              }))),
+              PropOrSpread::Prop(Box::new(Prop::KeyValue(KeyValueProp {
+                key: PropName::Ident(Ident::new("modules".into(), DUMMY_SP)),
+                value: Box::new(Expr::Array(ArrayLit {
+                  elems: vec![Some(ExprOrSpread {
+                    expr: Box::new(Expr::Bin(BinExpr {
+                      span: DUMMY_SP,
+                      op: BinaryOp::Add,
+                      left: Box::new(Expr::Lit(Lit::Str(Str {
+                        value: file_from.into(),
+                        span: DUMMY_SP,
+                        kind: StrKind::Normal {
+                          contains_quote: false,
+                        },
+                        has_escape: false,
+                      }))),
+                      right: Box::new(Expr::Lit(Lit::Str(Str {
+                        value: import_specifier.unwrap(),
+                        span: DUMMY_SP,
+                        kind: StrKind::Normal {
+                          contains_quote: false,
+                        },
+                        has_escape: false,
+                      }))),
+                    })),
+                    spread: None,
+                  })],
+                  span: DUMMY_SP,
+                })),
+              }))),
+            ],
+          }));
+
+          let mut props = vec![PropOrSpread::Prop(Box::new(Prop::KeyValue(KeyValueProp {
+            key: PropName::Ident(Ident::new("loadableGenerated".into(), DUMMY_SP)),
+            value: generated,
+          })))];
+
+          if expr.args.len() == 2 {
```

If the user gave three or more arguments, this can be a silent error.